docs/wave.rst

```diff
@@ -0,0 +1,106 @@
+Acoustic Wave Modelling using Devito
+==================================================
+
+Consider the acoustic wave equation given in 3D:
+
+.. math::
+   m(x,y,z)\frac{\partial^2 u}{\partial t^2} + \eta(x,y,z)\frac{\partial u}{\partial t}-\frac{\partial^2 u}{\partial x^2}-\frac{\partial^2 u}{\partial y^2}-\frac{\partial^2 u}{\partial z^2}= 0
+
+   u(x,y,z,0) = 0
+
+   \frac{\partial u(x,y,z,t)}{\partial t}|_{t=0} = 0
+
+where :math:`\eta` represents damping, :math:`m` represents square slowness
+
+In this tutorial, the origin, spacing and true velocity are stored in instance of :obj:`IGrid()` called :obj:`model`.
+The gap between two consecutive timestep :obj:`dt` is :samp:`model.get_critical_dt()`.
+All the seismic data and initial settings like the coordinate of source,
+receiver coordinates are stored in an instance of :obj:`IShot()` called :obj:`data`::
+
+  from containers import IGrid, IShot
+
+  model = IGrid()
+  dimensions = (50, 50, 50)
+  model.shape = dimensions
+  origin = (0., 0.)
+  spacing = (20., 20.)
+
+  # True velocity
+  true_vp = np.ones(dimensions) + 2.0
+  true_vp[:, :, int(dimensions[0] / 2):int(dimensions[0])] = 4.5
+
+  model.create_model(origin, spacing, true_vp)
+
+  # Define seismic data.
+  data = IShot()
+
+  dt = model.get_critical_dt()
+
+  # Omit the code that generate time_series, receiver_coords, location
+
```

Comment:
Do you mean I shouldn't omit codes that generates time_series, receiver_coords and location? I thought it's non-related to the purpose of the tutorial.